### PR TITLE
Design and UX in web3

### PR DESCRIPTION
**Pull request by konopkja**
## Description

Introduction to research and design in web3.  This is supposed to be a one  stop repositorz of useful links for designers and researchers coming to web3.

[link](https://ethereumorgwebsitedev01-konopkjadesignanduxdocs.gatsbyjs.io/en/developers/docs/design-and-ux/) to preview

## Related Issue

https://github.com/ethereum/ethereum-org-website/issues/8955

### CODE REVIEW

**Comment by nloureiro** [comment]
@konopkja do have an idea where will this page be linked? from the dev docs left-side navigation?

**Comment by konopkja** [comment]
> @konopkja do have an idea where will this page be linked? from the dev docs left-side navigation?

either in foundational topics before its robust enough, or as a standalone

**Comment by nhsz** [comment]
Having this under `/developers` is a bit weird to me and will hurt discoverability for sure, maybe we should have a `/design` route? IMO needs a bit more discussion

**Comment by konopkja** [comment]
> Having this under `/developers` is a bit weird to me and will hurt discoverability for sure, maybe we should have a `/design` route? IMO needs a bit more discussion

i agree. In an ideal solution it should not be under devs, but currently I dont see any quick solution for this. @nhsz do you have any ideas? how do you imagine "/design" to be implemented in the navigation?

**Comment by nhsz** [comment]
> > Having this under `/developers` is a bit weird to me and will hurt discoverability for sure, maybe we should have a `/design` route? IMO needs a bit more discussion
> 
> i agree. In an ideal solution it should not be under devs, but currently I dont see any quick solution for this. @nhsz do you have any ideas? how do you imagine "/design" to be implemented in the navigation?

I see no need to rush this . Don't know what could be the best approach as we currently don't have this kind of resources on the site, but I'd say maybe a separated hub could work better, sounds like it could be its own epic, is this planned? cc @minimalsm

**Comment by konopkja** [comment]
There are no deadlines, but I also don't see any risks with publishing this in dev docs. There are a few benefits to having this live. (indexing, design communities pointing to this, designers reading... preliminary feedback was very good from the communities I contacted)  

There is no epic around this for foreseeable future, but if this gets traction I can see it as good evidence of having more commitment in the next quarters.

**Comment by nhsz** [comment]
> There are no deadlines, but I also don't see any risks with publishing this in dev docs. There are a few benefits to having this live. (indexing, design communities pointing to this, designers reading... preliminary feedback was very good from the communities I contacted)
> 
> There is no epic around this for foreseeable future, but if this gets traction I can see it as good evidence of having more commitment in the next quarters.

Not the best for SEO hosting this `/developers`, that's my main pushback. Have you considered a subsection under [`/guides`](https://ethereum.org/en/guides/)?

**Comment by nhsz** [comment]
> > 
> 
> "Guides" section is for different target audience: beginners learning how to use Ethereum. Devs actually is not that far from design, in some methodologies designers are considered to be a "dev" role too (eg in scrum you have 3 roles: PO, dev, scrum master ). If the main pushback is SEO, while I agree with the sentiment, I don't think its a big enough issue, also with that logic we already have foundational topics that should not be there. If anything for future restructuring, it helps to be aware of all different areas that should be considered.

Navigation and information hierarchy is a current issue we have, if we start to use `/developers` to host any slightly related content these issues are just going to keep increasing. As a long term solution, IMO it should have it's own hub, with proper categories and navigation. If not, I'd say at least should be hosted on a different route.

**Comment by konopkja** [comment]
> 

I dont think so, that page is for ethereum.org design contributors while this list is for the designers/product owners/ researchers in the ethereum ecosystem.

**Comment by corwintines** [comment]
I think this is fine to have in the developer docs. Developer can mean many things, and plenty of those docs dont have anything to do with development as it is. Overall, that section of the site needs its information architecture looked at and rethought, which I think would result in a design hub, developer hub, general docs, etc. But, at the moment I think its fine to bring in there. 

A note on this though that still needs to be done, we need to add this to the nav still, otherwise no one will be able to navigate to it. 

TLDR, I think this is fine to add in the developer docs for now, Q3 we should begin actually rethinking this IA of the docs though.

**Review by corwintines — APPROVED** [review]
